mappings/net/minecraft/client/render/item/HeldItemRenderer.mapping

```diff
@@ -2,37 +2,84 @@ CLASS net/minecraft/unmapped/C_csxqddlf net/minecraft/client/render/item/HeldIte
 	FIELD f_atnadhfo EAT_OR_DRINK_Y_ANGLE_MULTIPLIER F
 	FIELD f_aytscjcb FIRST_PERSON_MAP_TRANSLATE_X F
 	FIELD f_bafqoewr equipProgressMainHand F
-	FIELD f_capjsulu ARM_Z_ANGLE_MULTIPLIER F
+	FIELD f_beoyytfk EAT_OR_DRINK_Y_POSITION_SCALE F
+	FIELD f_beuognql EAT_OR_DRINK_EXPONENT D
+	FIELD f_capjsulu MAP_HAND_Z_ROTATION F
 	FIELD f_cwnbijdt FIRST_PERSON_MAP_SECOND_SCALE F
 	FIELD f_cyqeubeh EQUIP_OFFSET_TRANSLATE_X F
+	FIELD f_dnefrohb MAP_HANDS_HEIGHT_SCALE F
+	FIELD f_duvgyzhb ARM_PRESWING_X_ROTATION F
 	FIELD f_elzylypi FIRST_PERSON_MAP_TRANSLATE_Y F
+	FIELD f_etfnujkg MAP_HANDS_POSITION_Y F
+	FIELD f_ezchtcmf ARM_PRESWING_Y_ROTATION F
```

Comment:
Is this 'arm pre-swing y rotation'? The `PRESWING` is a bit confusing

Reply:
I got confused with this, I was going to suggest an underscore however that made the name seem a bit incorrect looking

Reply:
`ARM_PREROTATION_Y_OFFSET` is mojmap, but not sure whats best in this case